api/server.go

```diff
@@ -0,0 +1,38 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"go.uber.org/zap"
+)
+
+// Ensure that we implement the server interface
+var _ ServerInterface = (*Server)(nil)
```

Comment:
I don't think this line is necessary to implement the server interface. Since RatsdChares is defined below, that automatically implements the interface.

So, you can remove this line. Thanks!

Reply:
@cowbon Thanks for the PR, tom! I've completed the review and requested some changes. If you want to discuss it, I'm glad to setup a meeting. Cheers.

Reply:
The type check is not deemed necessary. I removed it in the latest version of the patchset.